### DISCUSSION TITLE
Trivy did not report on PYSEC-2023-135

### DISCUSSION THREAD

**Discussion by donnieelmore** [aquasecurity/trivy#5778] [False Detection]
### IDs

Trivy did not report on PYSEC-2023-135

### Summary
We had been using the awx-ee container available on quay and were pinned to the following version: https://quay.io/repository/ansible/awx-ee/manifest/sha256:8e342ee7fe0d1b413a0ca24f45a1d29f0bf31829cbcd90104175c086cfa5a3e9?tab=vulnerabilities

The "PYSEC-2023-135" CVE was not showing in Trivy results ran via GitLab CI.

Not a new finding but more concerned that this finding was not found.

### Reproduction Steps

```bash
Ran trivy scanning on https://quay.io/repository/ansible/awx-ee/manifest/sha256:8e342ee7fe0d1b413a0ca24f45a1d29f0bf31829cbcd90104175c086cfa5a3e9?tab=vulnerabilities via GitLab CI and the vulnerability was not reported.
```

### Target

Container Image

### Scanner

Vulnerability

### Target OS

CentOS Stream

### Debug Output

```bash
{"vulnerabilities":[],"remediations":[],"scan":{"scanner":{"id":"trivy","name":"Trivy","url":"https://github.com/aquasecurity/trivy/","vendor":{"name":"GitLab"},"version":"0.44.1"},"analyzer":{"id":"gcs","name":"GitLab Container Scanning","vendor":{"name":"GitLab"},"version":"6.4.1"},"type":"container_scanning","start_time":"2023-12-02T05:12:03","end_time":"2023-12-02T05:13:03","status":"success"},"version":"15.0.6"}
```

### Version

```bash
{"vulnerabilities":[],"remediations":[],"scan":{"scanner":{"id":"trivy","name":"Trivy","url":"https://github.com/aquasecurity/trivy/","vendor":{"name":"GitLab"},"version":"0.44.1"},"analyzer":{"id":"gcs","name":"GitLab Container Scanning","vendor":{"name":"GitLab"},"version":"6.4.1"},"type":"container_scanning","start_time":"2023-12-02T05:12:03","end_time":"2023-12-02T05:13:03","status":"success"},"version":"15.0.6"}
```

### Checklist

- [X] Read [the documentation regarding wrong detection](https://aquasecurity.github.io/trivy/dev/community/contribute/discussion/#false-detection)
- [X] Ran Trivy with `-f json` that shows data sources and confirmed that the security advisory in data sources was correct

**Comment by DmitriyLewen** [comment]
Hello @donnieelmore 
Thanks for your report!

Is your CI using the latest trivy-db?

I can correctly find `PYSEC-2023-135` ([CVE-2023-37920](https://github.com/advisories/GHSA-xqr8-7jwr-rhp7) now):
```bash
➜ trivy -d image quay.io/ansible/awx-ee@sha256:8e342ee7fe0d1b413a0ca24f45a1d29f0bf31829cbcd90104175c086cfa5a3e9 | grep "certifi"
...
│ certifi (METADATA)      │ CVE-2023-37920      │ HIGH     │ fixed    │ 2023.5.7          │ 2023.7.22      │ python-certifi: Removal of e-Tugra root certificate         │

```
